runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java

```diff
@@ -63,4 +63,15 @@ default boolean showOpponentsInMenu()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "onlyShowBossHealthOverlay",
+		name = "Only show boss health overlay",
+		description = "Hides this plugin's additional health overlay; only modifies the in-game setting 'boss health overlay'",
+		position = 4
+	)
+	default boolean onlyShowBossHealthOverlay()
```

Comment:
This should be inverted to "Show opponent info overlay" and default to true.

Reply:
Yes I think that is clearer, changed.